flink-tests/src/test/resources/new-stateful-broadcast-udf-migration-itcase-flink1.17-rocksdb-checkpoint/0193581b-bd07-4b9d-b5d3-a6c986164fa4

```diff
@@ -0,0 +1,362 @@
+# This is a RocksDB option file.
+#
+# For detailed file format spec, please refer to the example file
+# in examples/rocksdb_option_file_example.ini
+#
```

Comment:
Do we really need RocksDB option files here?

// @rkhachatryan as Matthias said you probably should know

Reply:
@fredia can you give some insights into the rocksdb config file mentioned by @snuyanzin ? Is this expected content that's usually generated when creating checkpoints?

Reply:
@XComp Yes, rocksdb config file is usually generated when creating checkpoints.
Rocksdb checkpoint/savepoint-native is added after https://issues.apache.org/jira/browse/FLINK-26146, but some rocksdb configuration does not take effect, so no rocksdb config file is generated, see https://issues.apache.org/jira/browse/FLINK-26176.

The following are some existing checkpoint/native-savepoint files related to rocksdb. We can see that there were rocksdb config files in the past.

- [ ] new-stateful-broadcast-udf-migration-itcase-flink1.15-rocksdb-checkpoint  
- [ ] new-stateful-broadcast-udf-migration-itcase-flink1.16-rocksdb-checkpoint 
- [ ] new-stateful-udf-migration-itcase-flink1.15-rocksdb-checkpoint 
- [x] [new-stateful-udf-migration-itcase-flink1.16-rocksdb-checkpoint ](https://github.com/apache/flink/blob/master/flink-tests/src/test/resources/new-stateful-udf-migration-itcase-flink1.16-rocksdb-checkpoint/0e40bd9a-a3b3-4d71-9d32-dfaacba11244)
- [ ] stateful-scala-udf-migration-itcase-flink1.15-rocksdb-checkpoint 
- [ ] stateful-scala-udf-migration-itcase-flink1.16-rocksdb-checkpoint 
- [ ] stateful-scala-with-broadcast-udf-migration-itcase-flink1.15-rocksdb-checkpoint 
- [x] stateful-scala-with-broadcast-udf-migration-itcase-flink1.16-rocksdb-checkpoint 
- [x] type-serializer-snapshot-migration-itcase-flink1.16-rocksdb-checkpoint 


- [ ] new-stateful-broadcast-udf-migration-itcase-flink1.15-rocksdb-savepoint-native 
- [x] new-stateful-broadcast-udf-migration-itcase-flink1.16-rocksdb-savepoint-native 
- [ ] new-stateful-udf-migration-itcase-flink1.15-rocksdb-savepoint-native  
- [x] new-stateful-udf-migration-itcase-flink1.16-rocksdb-savepoint-native 
- [ ] stateful-scala-udf-migration-itcase-flink1.15-rocksdb-savepoint-native 
- [x] stateful-scala-udf-migration-itcase-flink1.16-rocksdb-savepoint-native 
- [ ] stateful-scala-with-broadcast-udf-migration-itcase-flink1.15-rocksdb-savepoint-native  
- [x] stateful-scala-with-broadcast-udf-migration-itcase-flink1.16-rocksdb-savepoint-native 
- [x] type-serializer-snapshot-migration-itcase-flink1.16-rocksdb-savepoint-native

Reply:
Thanks for clarification. @snuyanzin can we finalize the PR in that case?